### DISCUSSION TITLE
Include file in sql editor

### DISCUSSION THREAD

**Discussion by ClodoaldoPinto** [dbeaver/dbeaver#15747] [Q&A] [answered]
Is there a way to include a text file in a sql file in the sql editor? My use case is that I have a very long static CTE (a virtual table) that will be incorporated in many scripts. I have no create permission in the database so my only solution is the CTE.

**Comment by ShadelessFox** [comment, marked as answer]
Hello @ClodoaldoPinto,

This feature is already present in DBeaver.

You can use `@include filename` statement.

Please refer to our documentation for more information: https://dbeaver.com/docs/wiki/Client-Side-Scripting
